projects/Mallard/src/components/icons/Quote.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Svg, { Path } from 'react-native-svg'
+
+const Quote = ({ fill = '#FF7F0F', height = 22, width = 35 }) => (
```

Comment:
I'm assuming there isn't an existing icon for this in the Guardian icons font?

Reply:
I checked with @walaura and she said not.